### DISCUSSION TITLE
My hostname not working

### DISCUSSION THREAD

**Discussion by mrkingsleyobi** [postalserver/postal#1717] [Help with using Postal]
Try to get access to my postal server via the hostname i get error 502
![Screen Shot 2021-11-18 at 12 10 47 PM](https://user-images.githubusercontent.com/4938148/142405877-5ef21305-95d7-4113-9f0f-2506bda3f514.png)
But the Ip access works even after updated the DNS on the host
![Screen Shot 2021-11-18 at 12 10 29 PM](https://user-images.githubusercontent.com/4938148/142405955-00b86170-3c65-4029-b33e-933e871870b4.png)
Here is the status of my server
![Screen Shot 2021-11-18 at 12 10 03 PM](https://user-images.githubusercontent.com/4938148/142406006-be06d8af-306a-4d5e-9b7b-80a51a029145.png)
![Screen Shot 2021-11-18 at 12 10 17 PM](https://user-images.githubusercontent.com/4938148/142406016-6344a79d-45a2-4fe3-89c3-0d4ff0be193f.png)
I need help.

**Comment by willpower232** [comment]
502 means that nginx or caddy has not been able to route HTTP traffic to its destination (i.e. Postal)

It looks like you have resolved the issue however we wouldn't recommend exposing port 5000 to the internet as this is handled by nginx or caddy or whatever.
